src/calculateRentalCost.js

```diff
@@ -5,6 +5,16 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
+
+  return totalCost;
```

Comment:
The function name 'calculateRentalCost' is descriptive and clearly states its purpose.